specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/examples/WorkspaceVirtualNetworkPeeringCreate.json

```diff
@@ -0,0 +1,64 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "workspaceName": "myWorkspace",
+    "peeringName": "vNetPeeringTest",
+    "api-version": "2018-04-01",
+    "VirtualNetworkPeeringParameters": {
+      "properties": {
+        "allowForwardedTraffic": null,
+        "allowGatewayTransit": null,
+        "allowVirtualNetworkAccess": true,
+        "databricksVirtualNetwork": {
+          "id": "subscriptions/0140911e-1040-48da-8bc9-b99fb3dd88a6/resourceGroups/databricks-rg-adbworkspace-2jsxhmzoyooxm/providers/Microsoft.Network/virtualNetworks/workers-vnet"
+        },
+        "databricksAddressSpace": {
+          "addressPrefixes": [
+            "10.139.0.0/16"
+          ]
+        },
+        "remoteVirtualNetwork": {
+          "id": "/subscriptions/0140911e-1040-48da-8bc9-b99fb3dd88a6/resourceGroups/subramantest/providers/Microsoft.Network/virtualNetworks/subramanvnet"
+        },
+        "remoteAddressSpace": null,
+        "useRemoteGateways": null,
+        "peeringState": null,
+        "provisioningState": null
```

Comment:
Does a user need to input `peeringState` and `provisioningState`?

Reply:
No, you can send with null, our UI also sends null and in response we get actual state to which they are being updated on service side.

Reply:
this is marked as readonly in the spec, the example request should not include the property for correctness

---
In reply to: [433548227](https://github.com/Azure/azure-rest-api-specs/pull/9657#discussion_r433548227) [](ancestors = 433548227)